Jenkinsfile

```diff
@@ -36,7 +36,7 @@ pipeline {
         axes {
           axis {
             name 'NODE_VERSION'
-            values '10', '12', '14'
+            values '10', '12', '14', '16', '17'
```

Comment:
Are we able to drop 10 and 12?

Reply:
we could, but it's public, so technically we still support older versions.